components/mailchimp/sources/common/timer-based.js

```diff
@@ -0,0 +1,66 @@
+const base = require("./base");
+
+module.exports = {
+  ...base,
+  props: {
+    ...base.props,
+    // eslint-disable-next-line pipedream/props-label, pipedream/props-description
+    timer: {
```

Comment:
I don't think this commented line is needed for `timer`.
```suggestion
    timer: {
```